src/metabase/search/config.clj

```diff
@@ -41,8 +41,14 @@
   10)
 
 (def ^:const view-count-scaling
-  "The larger this value, the longer it will take for the score to approach 1.0. It will never quite reach it."
-  50)
+  "A constant factor influencing how quickly the incremental score grows with view count for a given search model.
+  The larger this value, the longer it will take for the score to approach 1.0. It will never quite reach it."
+  2)
+
+(def ^:const view-count-scaling-percentile
+  "The percentile of the given search model's view counts, to be multiplied by [[view-count-scaling]].
+  The larger this value, the longer it will take for the score to approach 1.0. It will never quite reach it."
+  0.9)
```

Comment:
This is a cut-off percentile, right? I'm thinking maybe a phrasing can be more clear, like "cut-off" and "outliers" used in there? 😁

Reply:
It's not quite a cut-off, as scores will continue to increase above this threshold. Let's discuss in our meeting later if we can find a better name, but I'll merge for now so we can experiment with it in practice first.